lms/security.py

```diff
@@ -112,7 +112,7 @@ def get_policy(path: str):
                 partial(get_lti_user_from_bearer_token, location="headers")
             )
 
-        if path in {"/assignment"}:
+        if path in {"/assignment", "/assignment/edit"}:
```

Comment:
Explict security policy for the new endpoint.

Reply:
:+1: